src/inputs/CheckmarksMultipleSelect/CheckmarksMultipleSelect.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Cosmo Tech.
```

Comment:
`CheckmarksMultipleSelect` could be designed as a generic component, without mentioning roles. Could we rename the following props ?
- `roles` -> `options`
- `agentRoles` -> `preselectedOptions`
- `onRoleChecked` -> `onSelect`
(`grantedRoles` state should be renamed too :slightly_smiling_face: )

A "remove access" button is too specific for a generic component, but we could use an `actionButtons` prop, of type `Array`, to let developers add as many extra actions as they wish. This new prop would replace the props `isRemoveAccessAvailable`, `removeAccessLabel` and `onRemoveAccess`. These extra actions would then be displayed after the `Divider` with the `.map()` method (if the array is not empty).